GliaWidgets/SecureConversations/SecureConversations.FileUploadList.swift

```diff
@@ -0,0 +1,965 @@
+import UIKit
+#if canImport(QuickLookThumbnailing)
+  import QuickLookThumbnailing
+#endif
+
+private extension Int {
+    static let maxUnscrollableViewsOnDefaultContentSizeCategory = 3
+    static let maxUnscrollableViewsOnLargeContentSizeCategory = 2
+}
+
+extension SecureConversations {
+    class FileUploadListView: UIView {
```

Comment:
I think we can inherit from `BaseView` and use `setup` & `defineLayout` functions, does it make sense?
```suggestion
    final class FileUploadListView: BaseView {
```